molgenis-web/src/main/java/org/molgenis/web/PluginController.java

```diff
@@ -58,6 +60,16 @@ public Entity getPluginSettings()
 		return RunAsSystemAspect.runAsSystem(() -> getPluginSettings(entityTypeId));
 	}
 
+	/**
+	 * Returns the description of the plugin
+	 */
+	public String getDescription(String pluginId)
+	{
+		Plugin plugin = RunAsSystemAspect.runAsSystem(
```

Comment:
If you are not allowed to read a Plugin entity, then you should not be allowed to read the description. Why runAsSystem?